pagopa/io-app
# chore(IT Wallet): [SIW-1510] Add European Health Insurance Card to the credential list

### ⚠️ Depends on https://github.com/pagopa/io-app/pull/6139

## Short description
This PR adds the European Health Insurance Card to the credential list in the onboarding screen

## List of changes proposed in this pull request
- Added `CredentialType.EUROPEAN_HEALTH_INSURANCE_CARD` to `availableCredentials`

## How to test
From the wallet screen tap on `Add to wallet` CTA, check that the European Health Insurance Card button is present.
